monitor/views.py

```
@@ -0,0 +1,177 @@
+"""DRF views for monitoring API."""
+
+from datetime import datetime
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)